stream/stream_bluray.c

```diff
@@ -552,15 +552,17 @@ static bool check_bdmv(const char *path)
         return false;
 
     char data[50] = {0};
```

Comment:
We can skip the init now.
```suggestion
    char data[50];
```